### DISCUSSION TITLE
@walletconnect/ethereum-provider extraneous-dependencies

### DISCUSSION THREAD

**Discussion by jeannemtl** [WalletConnect/walletconnect-monorepo#3599] [Developer Support]
I have the following installed: 		
"@walletconnect/ethereum-provider": "^2.10.0",
"@walletconnect/modal": "^2.6.1",
"@walletconnect/utils": "^2.8.0",
Which works fine locally except for this error:

'@walletconnect/ethereum-provider' should be listed in the project's dependencies. Run 'npm i -S @walletconnect/ethereum-provider' to add iteslint[import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-extraneous-dependencies.md)

Which I believe is the reason I am unable to build it on my remote code base.

Any thoughts on this?

**Comment by glitch-txs** [comment]
Hi, could you provider a [MRE](https://github.com/orgs/WalletConnect/discussions/3457)?
